templates/zerver/help/analytics.md

```diff
@@ -0,0 +1,83 @@
+# Analytics
+
+Zulip provides analytics for tracking various usage statistics about you and
+your organization.
+So, you can get a holistic view of your organization's communications.
```

Comment:
More formal wording: This will allow you to have a holistic view of your organization's communications.